ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Ds3Cli.java

```diff
@@ -79,6 +79,9 @@ public Ds3Cli(final Ds3Provider provider, final Arguments args, final FileUtils
         cliViews.put(CommandValue.PERFORMANCE,            new com.spectralogic.ds3cli.views.cli.PerformanceView());
         cliViews.put(CommandValue.GET_PHYSICAL_PLACEMENT, new com.spectralogic.ds3cli.views.cli.GetPhysicalPlacementWithFullDetailsView());
         cliViews.put(CommandValue.GET_TAPE_FAILURE,       new com.spectralogic.ds3cli.views.cli.GetTapeFailureView());
+        cliViews.put(CommandValue.GET_DATA_POLICIES,      new com.spectralogic.ds3cli.views.cli.GetDataPoliciesView());
+        cliViews.put(CommandValue.GET_DATA_POLICY,        new com.spectralogic.ds3cli.views.cli.GetDataPoliciesView());
```

Comment:
I put the single one in a list and pass it in so that the same view could be reused. There are two constructors in the Result: one takes the single and one takes the list.